offworld_gym/envs/gazebo_docker/remote_env.py

```diff
@@ -74,20 +77,28 @@ def _heart_beat_to_container_worker(grpc_port, weak_ref_to_parent_env):
     grpc_stub = RemoteEnvStub(channel)
     ever_made_successful_hearbeat = False
     hb_loop_start_time = time.time()
-    while True:
+    attempts = 0 
+    while attempts < 3:
+        logger.debug(f"\n Attempts to call client heartbeat service {attempts} times.")
         if weak_ref_to_parent_env() is None:
             # exit if parent env is garbage collected or otherwise deleted
             logger.debug("heartbeat thread exiting after parent env was destroyed.")
             return
         try:
-            grpc_stub.HeartBeat(Empty(), timeout=1.0)
+            before = time.time() 
+            grpc_stub.HeartBeat(Empty(), timeout=1.0) # if interupted, change timeout from 1.0 to 10.0
```

Comment:
I think we should set it a bit higher than 1.0.
Perhaps 3.0

Reply:
Also there is an extra comment that does not hold anymore

Reply:
will do